src/dispatcher.c

```diff
@@ -208,7 +208,7 @@ void *run_simulator(void *simulator_payload) {
         run_for(simulation, sp->step_cutoff);
         break;
     }
-    simulation_history_to_file(simulation);
+    history_to_file(simulation);
```

Comment:
I would keep the longer name here. C doesn't have namespaces so the longer you can make your variable names, the better